index.html

```diff
@@ -4,24 +4,56 @@
 <head>
 <meta charset="utf-8">
 <link rel="stylesheet" href="index.css">
-
-<!-- development version, includes helpful console warnings -->
-<script src="https://cdn.jsdelivr.net/npm/vue@2/dist/vue.js"></script>
-
 <body>
+
 <main id="chargen">
+
     <h1>CharGen</h1>
-    <div>
+
+    <div id="Info" class="clear">
+        <h2>Info</h2>
+        <div is="info-category"
+	     v-for="(item, index) in form.info"
+             v-bind:group_id="'info'"
+             v-bind:category="item"
+             v-bind:category_index="index"
+	></div>
+    </div>
+<!--             v-bind:key="item.id"-->
+
+    <div id="Attributes" class="clear">
         <h2>Attributes</h2>
-        <attribute-category 
-            v-for="item in Attributes"
-            v-bind:category="item"
-            v-bind:key="item.id"
-            v-on:sumchanged="RecomputeAttributePriorities()"
-        >
-        </attribute-category>
+        <div is="attribute-category"
+	     v-for="(item, index) in form.attributes"
+             v-bind:category="item"
+             v-on:sumchanged="RecomputeAttributePriorities()"
+	></div>    
+
     </div>
+
+    <div id="skills" class="clear">
+        <h2>Skills</h2>
+        <div is="attribute-category"
+	     v-for="(item, index) in form.skills"
+             v-bind:category="item"
+             v-on:sumchanged="RecomputeAttributePriorities()"
+	></div>    
+    </div>
+
+    <div id="spells" class="clear">
+        <h2>Spells</h2>
+        <div is="attribute-category"
+	     v-for="(item, index) in form.spells"
+             v-bind:category="item"
+             v-on:sumchanged="RecomputeAttributePriorities()"
```

Comment:
if we use this more then once, how many of categories are computed after one attribute/skill change ?